### PR TITLE
Layout Fix

### PR DESCRIPTION
**Pull request by steffandsv**
Correction of text and object positions on movie details page

### CODE REVIEW

**Comment by steffandsv** [comment]
# Before
before

# After
after

File: src/index.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ let CloseButton = (props) => (props)
 let goBack = () => ('/')
 
 let LegendedButton = ({children, title}) => (
-    <div style={{display: 'flex', flexDirection: 'column'}}>
+    <div style={{display: 'flex', flexDirection: 'row'}}>
```
Comment (btzr-io) [inline src/index.js:14]: @xaiki why are we using inline styles?

**Comment by btzr-io** [comment]
Looks good ( Still a lot of work to do here :upside_down_face:  )
My only complain is the `inline` styles.

**Comment by btzr-io** [comment]
@xaiki I'll merge this unless you have something to `say / review`, 
just let me know. :v:

File: src/index.js (hunk 1 above)
Comment (xaiki) [reply src/index.js:14]: because we're waiting on someone that «actually knows what he's doing» to come and fix it =)

**Pull request merged**